scripts/build.sh

```diff
@@ -17,8 +17,14 @@ BASEDIR=$(dirname $(readlink -f $(dirname $0)))
 
  for d in opentelemetry-api/ opentelemetry-sdk/ ext/*/ ; do
    (
+     echo "building $d"
      cd "$d"
-     python3 setup.py --verbose bdist_wheel --dist-dir "$BASEDIR/dist/"
+     # some ext directories (such as docker tests)
+     # are not intended to be packaged. Verify the
+     # intent by looking for a setup.py
```

Comment:
Note that this still builds `opentelemetry-ext-testutil-0.4.dev0.tar.gz`.

Reply:
should we not publish testutil? I'm not sure the right market to set in that case, although maybe we should just move to a separate directory?

Reply:
Actually it seems like we should publish it, as the extensions need it to run tests, and it'd be frustrating I think to not be able to not be able to run tests without installing a package that only exists in a source tree.